code/controllers/subsystem/persistence.dm

```diff
@@ -136,6 +137,26 @@ SUBSYSTEM_DEF(persistence)
 		saved_trophies = json["data"]
 	SetUpTrophies(saved_trophies.Copy())
 
+/datum/controller/subsystem/persistence/proc/LoadRecentModes()
+	if(fexists("data/RecentModes.sav")) //legacy compatability to convert old format to new
```

Comment:
There is no legacy saves that exist for this since you're creating it now. You can remove the .sav stuff